core/src/test/java/io/grpc/internal/ClientCallImplTest.java

```diff
@@ -156,7 +158,7 @@ public void advertisedEncodingsAreSent() {
     call.start(callListener, new Metadata());
 
     ArgumentCaptor<Metadata> metadataCaptor = ArgumentCaptor.forClass(Metadata.class);
-    verify(transport).newStream(eq(method), metadataCaptor.capture());
+    verify(transport).newStream(eq(method), metadataCaptor.capture(), same(CallOptions.DEFAULT));
```

Comment:
We will need a new test method for the propagation of `CallOptions` to `newStream()`.